quiz.rb

```diff
@@ -2,19 +2,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  attr_reader :name, :breed, :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
+
+
 # Question 2
 # Create a new Dog object.
 
+fluffy = Dog.new("fluffy", "lab", 2)
+
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
+class Person
+  def initialize(age)
+    @age = age
```

Comment:
Almost! By default, all instance variables are totally private. `attr_accessor` allows us to read from, and write to, both `name` and `location`, but we need `attr_reader` to have read access to `age`.